pkg/webhook/inject/inject.go

```diff
@@ -137,55 +155,55 @@ func Inject(res *v1.AdmissionRequest, cli client.Client, cfg *config.Config, con
 }
 
 // Check whether the target resource need to be injected and return the required config name
-func injectRequired(metadata *metav1.ObjectMeta, cli client.Client, cfg *config.Config, controllerCfg *controllerCfg.ChaosControllerConfig) (string, bool) {
+func (i Injector) injectRequired(metadata *metav1.ObjectMeta, cli client.Client, cfg *config.Config, controllerCfg *controllerCfg.ChaosControllerConfig) (string, bool) {
```

Comment:
ditto, please use pointer receiver instead of value receiver.

please use fields inside of `Injector` instead of acquiring them as method parameters